web/satellite/src/components/header/projectSelection/ProjectSelectionDropdown.vue

```diff
@@ -42,8 +42,12 @@
             this.$store.dispatch(APP_STATE_ACTIONS.TOGGLE_PROJECTS);
             this.$store.dispatch(PM_ACTIONS.SET_SEARCH_QUERY, '');
 
-            // TODO: add types
-            const usageResponse = await this.$store.dispatch(PROJECT_USAGE_ACTIONS.FETCH_CURRENT_ROLLUP);
+            try {
+                await this.$store.dispatch(PROJECT_USAGE_ACTIONS.FETCH_CURRENT_ROLLUP);
+            } catch (err) {
+                await this.$store.dispatch(NOTIFICATION_ACTIONS.ERROR, `Unable to fetch project usage. ${err.message}`);
```

Comment:
Do we need to wait for this action?

Reply:
all store actions are async

Reply:
Why is it all async? I thought it should only be async if we are making a network call. For example, on line 56 in this file, the notification_actions is not waited